tests/21-ct-clean-up.sh

```diff
@@ -107,6 +121,21 @@ function test_unreachability(){
   fi
 }
 
+function test_reachability_port(){
+  local DROP_TIMEOUT="2"
+
+  local container="${1}"
+  local dst_ip="${2}"
+  local dst_port="${3}"
+  local http_path="${4}"
+  log "trying to curl http://${dst_ip}:${dst_port}${http_path} from client container (shouldn't work)"
+  log "docker exec -d "${container}" bash -c \"curl -s -o /dev/null -w \"%{http_code}\" --connect-timeout $DROP_TIMEOUT -XGET http://${dst_ip}:${dst_port}${http_path}\""
+  ret="$(docker exec -d "${container}" bash -c "curl -s -o /dev/null -w \"%{http_code}\" --connect-timeout $DROP_TIMEOUT -XGET http://${dst_ip}:${dst_port}${http_path}")"
```

Comment:
I think you can remove the quotes before and after `${container}`

Reply:
Done